spinoso-time/src/time/tzrs/offset.rs

```diff
@@ -185,6 +185,13 @@ impl From<&str> for Offset {
     }
 }
 
+impl From<&[u8]> for Offset {
+    fn from(input: &[u8]) -> Self {
+        let input = str::from_utf8(input).expect("Invalid UTF8");
```

Comment:
`From` impls should never panic. This one should probably be a `TryFrom` impl, but we can defer fixing that until the errors PR.

Reply:
Ah yes. It's consistently incorrect with this implementation - rather than inconsistently incorrect :sweat_smile: